deployments/kovan/migrations/1653512186_seed_rewards_with_comp.ts

```diff
@@ -10,64 +11,65 @@ migration<Vars>('1653512186_seed_rewards_with_comp', {
     return {};
   },
   enact: async (deploymentManager: DeploymentManager, vars: Vars) => {
-    const signer = await deploymentManager.getSigner();
+    const { ethers } = deploymentManager.hre;
 
     const timelock = await deploymentManager.contract('timelock') as SimpleTimelock;
     const governor = await deploymentManager.contract('governor') as GovernorSimple;
     const comet = await deploymentManager.contract('comet') as CometInterface;
     const rewards = await deploymentManager.contract('rewards') as CometRewards;
     const COMP = await deploymentManager.contract('COMP') as ERC20;
 
-    console.log('Governor ', governor.address)
-    console.log('Timelock ', timelock.address)
-    console.log('Rewards ', rewards.address)
-
     const timelockCompBalance = await COMP.balanceOf(timelock.address);
-    console.log('COMP balance of Timelock: ', timelockCompBalance);
+    debug(`COMP balance of Timelock: ${timelockCompBalance}`);
 
     // Steps:
     // 1. Set reward config in CometRewards.
     // 2. Send half of the Timelock's COMP to CometRewards.
     const setRewardConfigCalldata = ethers.utils.defaultAbiCoder.encode(["address", "address"], [comet.address, COMP.address]);
     const transferCompCalldata = ethers.utils.defaultAbiCoder.encode(["address", "uint"], [rewards.address, timelockCompBalance.div(2)]);
 
-    const governorAsAdmin = governor.connect(signer);
-
     // Create a new proposal and queue it up. Execution can be done manually or in a third step.
-    let tx = await (await governorAsAdmin.propose(
-      [
-        rewards.address,
-        COMP.address,
-      ],
-      [
-        0,
-        0,
-      ],
-      [
-        "setRewardConfig(address,address)",
-        "transfer(address,uint256)",
-      ],
-      [
-        setRewardConfigCalldata,
-        transferCompCalldata,
-      ],
-      'Set RewardConfig and transfer COMP to CometRewards')
-    ).wait();
-    let event = tx.events.find(event => event.event === 'ProposalCreated');
-    let [proposalId] = event.args;
+    const txn = await deploymentManager.asyncCallWithRetry(
+      async (signer_) => (await governor.connect(signer_).propose(
+        [
+          rewards.address,
+          COMP.address,
+        ],
+        [
+          0,
+          0,
+        ],
+        [
+          "setRewardConfig(address,address)",
+          "transfer(address,uint256)",
+        ],
+        [
+          setRewardConfigCalldata,
+          transferCompCalldata,
+        ],
+        'Set RewardConfig and transfer COMP to CometRewards')
+      ).wait()
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    await deploymentManager.asyncCallWithRetry(
```

Comment:
I wonder if we should give this fn a friendly name like `do` since its going to be used everywhere

Reply:
Yeah, `asyncCallWithRetry` is kind of long. I can do a refactor PR. I'm thinking `do`, `retry`, or `doRetry`.